recipes/hist/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hist" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hist-{{ version }}.tar.gz
+  sha256: 48edff8f498ea7c5c6d8057c6f001a2ce394acd0923b921c56b76fb732e11df3
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - hist=hist.classichist:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - boost-histogram ~=0.11.0
+    - histoprint >=1.4
+    - numpy >=1.13.3
+    - python
+    - typing-extensions  # [py<38]
```

Comment:
I'm wondering if we can remove the py version restriction if this falls back to a reasonable choice of builtins when using py3.8. The package recipe lists this as `noarch: python`

Reply:
It does.